### PR TITLE
Use extension built within Dockerfile instead of from the local system

### LINKED ISSUE #508: Make dockerfile work from unconfigured repo
For some reason running ```build-extension.sh``` was required to build a proper docker image from the Dockerfile.
From what I can tell the only difference between the Dockerfile and the script is the ```cp dist/*.zip ./openwpm.xpi```at the end of the script.

### PR DESCRIPTION
**Pull request by vringar**
Made the Dockerfile do the same steps as build-extension.sh
Hopefully closes #508

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ COPY automation/Extension/firefox/. ./
 COPY automation/Extension/webext-instrumentation/. ../webext-instrumentation
 RUN npm install
 RUN npm run build
+RUN cp dist/openwpm-1.0.zip openwpm.xpi
```
Comment (englehardt) [inline Dockerfile]: I think we should use `COPY` here, instead of `RUN cp`?

Comment (vringar) [reply Dockerfile]: `COPY` moves files from the host system, while `RUN cp` copies filed around inside the image.
At least this is my understanding after reading the documentation

Comment (vringar) [reply Dockerfile]: Thus far we seem to have accidentally copied in the extension build on the host and never used the extension built inside of the image.

Comment (englehardt) [reply Dockerfile]: Makes sense. Good work finding this!

**Pull request merged**